main_app/models.py

```diff
@@ -46,3 +46,64 @@ class UserModel(PermissionsMixin, AbstractBaseUser):
 
     def __str__(self):
         return self.email
+
+
+class MustModel(models.Model):
+    game_id = models.IntegerField()
+    user = models.ForeignKey(UserModel, on_delete=models.CASCADE)
+    is_deleted = models.BooleanField(default=False)
+
+    def __str__(self):
+        return f"{self.user.username} {self.game_id}"
+
+
+class GameModel(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=100)
+    rating = models.DecimalField(decimal_places=0, max_digits=3, null=True)
+    version_title = models.CharField(max_length=100, null=True)
+    aggregated_rating = models.DecimalField(decimal_places=0, max_digits=3, null=True)
+    summary = models.CharField(max_length=2500, null=True)
+    first_release_date = models.DateField(null=True)
+    rating_count = models.DecimalField(decimal_places=0, max_digits=3, null=True)
+    aggregated_rating_count = models.DecimalField(decimal_places=0, max_digits=3, null=True)
+
+
+class GenreModel(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=50)
+
+
+class PlatformModel(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=50)
+
+
+class ScreenshotModel(models.Model):
+    id = models.AutoField(primary_key=True)
+    url = models.URLField()
+
+
+class CoverModel(models.Model):
+    id = models.AutoField(primary_key=True)
+    url = models.URLField()
```

Comment:
Let's replace 2 models: Cover and Screenshot with just single one as the're almost the same
```
class Image(models.Model):
    url = models.URLField()
    type = models.ChoiceField() # screenshot or cover
    game = models.ForeignKey(Game, related_name="images", on_delete=models.CASCADE)
```
hence `CoverGameModel` and `ScreenshotGameModel` not needed here also